securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -54,6 +54,29 @@ def _journalist_logs_in(self):
         headline = self.driver.find_element_by_css_selector('span.headline')
         assert 'Sources' in headline.text
 
+    def _journalist_visits_col(self):
+        self.driver.find_element_by_css_selector(
+            '#un-starred-source-link-1').click()
+
+    def _journalist_select_first_doc(self):
+        self.driver.find_elements_by_name('doc_names_selected')[0].click()
+
+    def _journalist_delete_one_javascript(self):
```

Comment:
It's not super clear what this function does from the name, maybe `_journalist_clicks_delete_selected(self)`?